### PR TITLE
Changes to make gtk2 and gtk3 match each other.

### PR DESCRIPTION
**Pull request by TiZ-EX1**
Exactly what it says on the tin. Certain looks from gtk2 were chosen over gtk3, and vice versa. Also, I took some artistic liberty in some places. Active notebook uses base_color again like in Numix, for one.

### CODE REVIEW

**Comment by TiZ-EX1** [comment]
Also, this is only <= 3.18. I've elected to not touch 3.20 with a 40-foot stick. :(

File: gtk-3.0/assets/checkbox-checked-dark.svg [hunk 1]
```diff
@@ -2,7 +2,9 @@
...
+<path style=" stroke:none;fill-rule:nonzero;fill:#000000;fill-opacity:0.20;" d="M 16.667969 0 C 7.421875 0 0 7.421875 0 16.667969 L 0 116.667969 C 0 125.910156 7.421875 133.332031 16.667969 133.332031 L 116.667969 133.332031 C 125.910156 133.332031 133.332031 125.910156 133.332031 116.667969 L 133.332031 16.667969 C 133.332031 7.421875 125.910156 0 116.667969 0 Z M 16.667969 8.332031 L 116.667969 8.332031 C 121.289062 8.332031 125 12.042969 125 16.667969 L 125 116.667969 C 125 121.289062 121.289062 125 116.667969 125 L 16.667969 125 C 12.042969 125 8.332031 121.289062 8.332031 116.667969 L 8.332031 16.667969 C 8.332031 12.042969 12.042969 8.332031 16.667969 8.332031 Z M 16.667969 8.332031 "/>
```
Comment (actionless) [inline gtk-3.0/assets/checkbox-checked-dark.svg]: here and next on in svg-s you are adding hardcoded color

File: gtk-2.0/gtkrc [hunk 2]
```diff
@@ -168,8 +168,8 @@ style "murrine-thin" {
 # Notebook
 
 style "clearlooks-notebook-bg" {
-	bg[NORMAL] = @bg_color
-	bg[ACTIVE] = shade (0.80, @bg_color)
+	bg[NORMAL] = @base_color
+	bg[ACTIVE] = shade (0.87, @base_color)
```
Comment (actionless) [inline gtk-2.0/gtkrc]: the idea to use base_color for notebook bg not seems logical to me -- it's usually used for user input, so it will be better to revert the change

Comment (TiZ-EX1) [inline gtk-2.0/gtkrc]: That's fine by me. It's ultimately your theme; it won't hurt my feelings for you to revert it. :)

File: gtk-3.0/assets/checkbox-checked-dark.svg (hunk 1 above)
Comment (TiZ-EX1) [inline gtk-3.0/assets/checkbox-checked-dark.svg]: Replacing it to FG causes it to look really bad with dark themes and terribly mismatch gtk2. What this thing is for... it's a duplicate of the path before it, and it's black and 20% opacity. Thus it darkens the color underneath it. It's difficult to arbitrarily darken or lighten colors in a script so I chose this method.

Comment (actionless) [inline gtk-3.0/assets/checkbox-checked-dark.svg]: isn't FG almost black?

i was thinking what for case of bg darker than `#333333` — in that case that will do less effect, while fg in that case is usually contrast to the current bg color so applying with 20% transparency will actually lighten it up

File: gtk-2.0/gtkrc (hunk 2 above)
Comment (actionless) [inline gtk-2.0/gtkrc]: could you please do that for both gtk versions once when you'll be rebasing (i think after we'll clarify about svg-s)?

File: gtk-3.0/assets/checkbox-checked-dark.svg (hunk 1 above)
Comment (TiZ-EX1) [inline gtk-3.0/assets/checkbox-checked-dark.svg]: Sure, using FG is okay for light themes, but even then it'll still look weird, because most widgets create their border by darkening the BG color.

Huh? Layering a transparent black onto something will never lighten it, it will always darken it. Sure, on dark themes it will have less effect, but that's consistent with the behavior of widget borders on dark themes anyways. Those borders are just shade(0.80, color); this is basically the same thing.

File: gtk-2.0/gtkrc (hunk 2 above)
Comment (TiZ-EX1) [inline gtk-2.0/gtkrc]: Sure thing dude. :)

File: gtk-3.0/assets/checkbox-checked-dark.svg (hunk 1 above)
Comment (actionless) [inline gtk-3.0/assets/checkbox-checked-dark.svg]: i mean what if bg is dark then fg will be light, let's say white.

so applying 20% white on top of something will lighten it up

Comment (TiZ-EX1) [inline gtk-3.0/assets/checkbox-checked-dark.svg]: I'm not applying 20% white, I'm applying 20% #000000. That's why it's hardcoded.

Comment (actionless) [inline gtk-3.0/assets/checkbox-checked-dark.svg]: i am still not getting, could you make the screenshots with dark and light themes, with hardcoded value and fg?

Comment (TiZ-EX1) [inline gtk-3.0/assets/checkbox-checked-dark.svg]: Sorry it took me so long to get around to doing this.

Here is original Numix:
numix
Here is original oomox with Numix color scheme:
oomox-numix-orig
Here is my fork with Numix color scheme:
oomox-numix-tiz
Here is original oomox with dark color scheme:
oomox-dark-orig
Here is my fork with dark color scheme:
oomox-dark-tiz

Do the changes make more sense when you see them in action? The checkboxes when rendered this way are much closer to original numix.

Comment (actionless) [inline gtk-3.0/assets/checkbox-checked-dark.svg]: in light colorscheme both look fine, in dark both not really (first too contrast, second otherwise) :)

have you tried the idea of 20% FG? i think it should bring the middle results usable for both

Comment (TiZ-EX1) [inline gtk-3.0/assets/checkbox-checked-dark.svg]: 20% FG will still end up with the checkbox border being light on a dark theme whereas every other widget will have a dark border. The contrast isn't good on dark, but that's going to be the case in every dark theme, what with the base theme being designed around a light color scheme. The official Numix Dark has the very same contrast issue.

For that matter, we keep using the word "issue"... when you're actually using a dark theme everywhere, this amount of contrast goes just fine with it, in my opinion.

At the end of the day, this is your fork. Whatever you decide to do is fine with me. :)

Comment (actionless) [inline gtk-3.0/assets/checkbox-checked-dark.svg]: yup, since the borders are made like that let's keep your change to checkboxes

dark theme issues could be fixed later on all together (because button borders have a similar issue too and so on)

**Comment by actionless** [comment]
ok, so please revert light bg for notebooks and we are good to go  🚀

**Comment by TiZ-EX1** [comment]
I deed it. Anything else I need to fix?

**Comment by actionless** [comment]
rebase on master to make merge button green

**Comment by TiZ-EX1** [comment]
Okay, I think it's good. Git does some things sometimes that confuse me, sorry about that, haha.

**Comment by actionless** [comment]
awesome job, thanks a lot!

**Comment by TiZ-EX1** [comment]
Thank you likewise!
